packages/cli/testSetup.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+jest.mock('./src/util/logger', () => ({
```

Comment:
actually, plain `jest.mock('./src/util/logger')` should be enough, because methods will be stubbed automatically :)

Reply:
done